org.scala-ide.sdt.debug/plugin.xml

```diff
@@ -28,7 +28,7 @@
             delegate="org.scalaide.debug.internal.launching.ScalaApplicationLaunchConfigurationDelegate"
             delegateDescription="The Scala JVM Launcher supports debugging of local Scala using the new Scala debugger"
             id="scala.application.new"
-            modes="debug"
+            modes="debug, run"
```

Comment:
I don't yet understand why we need this change. Want we be able to run the EE in both debug and run mode or is ScalaTest only available in run mode but somehow executed in debug mode?